src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ CHANGELOG
  * Deprecated passing an array as the second argument of the `RadioListMapper::mapDataToForms()` method, pass `\Traversable` instead.
  * Deprecated passing an array as the first argument of the `RadioListMapper::mapFormsToData()` method, pass `\Traversable` instead.
  * Added a `choice_translation_parameters` option to `ChoiceType`
+ * Added `UuidType` and `UlidType`
```

Comment:
```suggestion
 * Add `UuidType` and `UlidType`
```
And should be moved to the top 🔝

Reply:
`Added` is kinda consistent with other changelog entries. 🤔

Reply:
That's true, but @fabpot send a PR regarding the convention he likes to use for these entries earlier this week which I guess Oskar is referring to here.

Reply:
https://github.com/symfony/symfony-docs/pull/14830

Reply:
fixed